scvi/external/solo/_model.py

```diff
@@ -193,7 +193,7 @@ def from_scvi_model(
             doublet_adata.obs[LABELS_KEY] = "doublet"
 
             full_adata = latent_adata.concatenate(doublet_adata)
-            setup_anndata(full_adata, labels_key=LABELS_KEY)
+            _setup_anndata(full_adata, labels_key=LABELS_KEY)
```

Comment:
@adamgayoso  self or scvi_model? why self?

Reply:
because `full_adata` corresponds to the input on this model. Practically it shouldn't matter though, but if this model's setup anndata ever adds things, this will be correct